### ISSUE TITLE
Altair and other JS extensions do not work in Voila or Colab

### ISSUE THREAD

**Issue by maartenbreddels** [widgetti/solara#486]
In https://github.com/widgetti/solara/blob/master/solara/widgets/vue/vegalite.vue
we assume the `/_solara/_cdn/` endpoint can be reached. This endpoint exists in notebook, lab and solara server, but not in Voila (except when running as a voila extension in jupyter server) or colab.

A workaround for now is to manually set the `solara_cdn` javascript variable.
```python
import IPython.display
# make solara use a real cdn, since the solara cdn proxy is not available
display(IPython.display.Javascript("solara_cdn = 'https://cdn.jsdelivr.net/npm/'"))
```

**Comment by maartenbreddels** [comment]
Found this our while writing a test for https://github.com/widgetti/solara/pull/479

**Comment by iisakkirotko** [comment]
Also worth noting that the endpoint is available in VSCode

**Comment by maartenbreddels** [comment]
Altair should work now due to #487, other js libraries not yet

**Comment by Jhsmit** [comment]
For me Altair also doesnt work in notebooks

`404 GET /_solara/cdn/requirejs@2.3.6/require.js`

**Comment by maartenbreddels** [comment]
Should be fixed due to #479 

Keeping this open till we fix this for the other libraries.

**Comment by maartenbreddels** [comment]
Fixed in #763  (Solara 1.39.0)
